code/modules/mob/living/simple_animal/hostile/retaliate/goose.dm

```diff
@@ -158,10 +158,11 @@
 
 /mob/living/simple_animal/hostile/retaliate/goose/Life(seconds_per_tick = SSMOBS_DT, times_fired)
 	. = ..()
-	if(choking && !stat)
-		do_jitter_animation(50)
-		if(SPT_PROB(10, seconds_per_tick))
-			emote("gasp")
+	if(!choking || stat)
+		return
+	do_jitter_animation(50)
```

Comment:
50 what?

Reply:
`do_jitter_animation` is a goofy proc it's an arbitrary number